swagger/fs_wrapper.go

```diff
@@ -0,0 +1,95 @@
+package swagger
+
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"io/fs"
+	"strings"
+	"time"
+)
+
+type fsWrapper struct {
+	fs           fs.FS
+	jsonFileName string
```

Comment:
The spec itself might not be json-formatted, it could be yaml.
Could we replace the occurences of `jsonFileName` and `openApiJson` and `jsonFilePath` with something unique, such as `specPath`, or similar ?